docs/_extra/api/hypothesis.yaml

```diff
@@ -131,8 +131,18 @@ paths:
           description: Annotation not found or no permission to view
           schema:
             $ref: '#/definitions/Error'
-    put:
+    patch:
       summary: Update an annotation
+      description: |
+        This endpoint is available under both the `PATCH` and `PUT`
+        request methods. Both endpoints have PATCH-characteristics
```

Comment:
That reads more clearly to me, but that’s totally subjective personal taste. I don’t have strong feelings about this, so go with whichever version you prefer.